src/current/v24.3/manage-logical-data-replication.md

```diff
@@ -0,0 +1,195 @@
+    ----------------------+----------+--------------------------------+-------------------------------+-------------------------------+--------------------------+-----------------------------------------------------------------------------------------
+    1010959260799270913 | running  | {movr.public.promo_codes}      | 2024-10-24 17:50:05+00        | 2024-10-10 20:04:42.196982+00 | LWW                      | LOGICAL REPLICATION STREAM into movr.public.promo_codes from external://cluster_a
+    1014047902397333505 | canceled | {defaultdb.public.office_dogs} | 2024-10-24 17:30:25+00        | 2024-10-21 17:54:20.797643+00 | LWW                      | LOGICAL REPLICATION STREAM into defaultdb.public.office_dogs from external://cluster_a
+    ~~~
+
+1. Drop the LDR job on both clusters. Canceling the LDR streams will remove the history retention job, which will cause the data to be garbage collected according to the [`gc.ttlseconds`]({% link {{ page.version.version }}/configure-replication-zones.md %}#gc-ttlseconds) setting. Use [`CANCEL JOB`]({% link {{ page.version.version }}/cancel-job.md %}):
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CANCEL JOB {ldr_job_id};
+    ~~~
+
+1. Perform the schema change on cluster A. 
+1. Drop the table from cluster B. 
+1. Recreate the table and its new schema on cluster B. 
+1. Create new LDR streams for the table on both clusters A and B. Run `CREATE LOGICAL REPLICATION STREAM` from the **destination** cluster for each stream:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE LOGICAL REPLICATION STREAM FROM TABLE {database.public.table_name} ON 'external://{source_external_connection}' INTO TABLE {database.public.table_name};
+    ~~~
+
+#### Coordinate schema changes for unidirectional LDR 
+
+If you have a unidirectional LDR setup, you should cancel the running LDR stream and redirect all application traffic to the source cluster.
+
+1. Drop the LDR job on the **destination** cluster. Canceling the LDR job will remove the history retention job, which will cause the data to be garbage collected according to the [`gc.ttlseconds`]({% link {{ page.version.version }}/configure-replication-zones.md %}#gc-ttlseconds) setting. Use [`CANCEL JOB`]({% link {{ page.version.version }}/cancel-job.md %}):
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CANCEL JOB {ldr_job_id};
+    ~~~
+
+1. Once the job has `canceled`, perform the required schema change on both the source and destination tables.
+1. Start a new LDR job from the **destination** cluster:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    CREATE LOGICAL REPLICATION STREAM FROM TABLE {database.public.table_name} ON 'external://{source_external_connection}' INTO TABLE {database.public.table_name};
+    ~~~
+
+## Jobs and LDR
```

Comment:
we should call out that Backups function the same as changefeds. like they also contain their application writes, as well as the replicated writes, so u only need to run a backup job on one cluster.

Reply:
I've added a paragraph here on backups, hopefully Michael can check this as he reviews.